build/run-tests.sh

```diff
@@ -1,7 +1,25 @@
 #!/usr/bin/env bash
+set -e
 
-docker-compose build --no-cache
+function stop_containers() {
+  docker-compose down
+}
+
+trap stop_containers EXIT
+
+if [[ ! -d ../verify-event-system-database-scripts ]]; then
+    echo "Cloning database migration scripts repo..."
+    pushd ..
+    git clone git@github.com:alphagov/verify-event-system-database-scripts.git
+    popd
+fi
+
+echo "Building test imgages..."
+docker-compose build
+
+echo "Starting up database..."
```

Comment:
This might wanna be a separate PR, but let's talk tomorrow :-)

Reply:
This is actually part of https://github.com/alphagov/verify-event-recorder-service/pull/57